static/prototype/huron-assets/index.js

```diff
@@ -0,0 +1,86 @@
+/*eslint-disable*/
+
+
+var store = require('./huron-store');
+var InsertNodes = require('./insertNodes').default;
+var sectionTemplate = require('./section.hbs');
+var assets = require.context('../partials', true, /\.html|\.json|\.hbs$/);
+var modules = {};
+
+modules['./huron-assets/section.hbs'] = sectionTemplate;
+
+assets.keys().forEach(function(key) {
+  modules[key] = assets(key);
+});
+
+var insert = window.insert ? window.insert :
+  new InsertNodes(modules, store);
+window.insert = insert;
+
+if (module.hot) {
+  // Hot Module Replacement for huron components (json, hbs, html)
+  module.hot.accept(
+    assets.id,
+    () => {
+      var newAssets = require.context(
+        '../partials',
+        true,
+        /\.html|\.json|\.hbs$/
+      );
+      var newModules = newAssets.keys()
+        .map((key) => {
```

Comment:
Wait... arrow function. That's ES6.

Reply:
haha this is a built file

Reply:
Oh wait not it's not. Yeah, I'm not actually sure how to tackle this issue. It's JS that needs to be written programmatically, so I don't know how to lint or transpile it. It's like meta-js.